### DISCUSSION TITLE
How to find PR authors for 2022, 2023 and 2024?

### DISCUSSION THREAD

**Discussion by aliok** [cncf/devstats#66] [Q&A]
I want to check out the list of PR authors for 2022, 2023 and 2024 separately.

I couldn't find a dashboard for that. There are some dashboards that show PR authors, but they don't allow selecting a date range (only allow selecting last N months/years).

I can get the data for last 3 years and last years, substract them to find the PR author data for [t-3 years, y-2 years] but I am curious if there's any better way.

My eventual goal is to find out these:
- How many PR authors were there in 2022, 2023 and 2024?
- How many of them were employed by main vendors of Knative (Red Hat, Google, VMware, IBM, etc)
- How many of them were "passion" driven contributors?
- Percent of the PR authors for these criteria
- Percent of the number of PRs they've created (impact)

**Comment by lukaszgryglicki** [comment]
I can provide a custom data for you on my next CNCF working day.
I'll TAL on this as soon as I can.

**Reply by aliok** [reply]
Thanks a lot.

**Comment by lukaszgryglicki** [comment]
Ideally create an issue of type "feature request" for this, if not I can can create one with exact same description.

**Reply by aliok** [reply]
Let me create a ticket then.

I didn't want to create directly, because I don't know about the feasibility of the request.

**Reply by aliok** [reply]
https://github.com/cncf/devstats/issues/67

Thanks @lukaszgryglicki
